megaavr/cores/dxcore/ADC.h

```diff
@@ -16,12 +38,14 @@ typedef struct ADCConfig_struct {
     uint8_t convMode;    // 0 = single-ended, 1 = differential
     uint8_t muxNeg;      // Differential mode negative pin
     uint8_t resolution;  // ADC bit resolution
-    uint8_t prescaler;   // ADC clock prescaler
+    uint16_t prescaler;   // ADC clock prescaler
```

Comment:
I dont think the `uint16_t` is necessary. The CTRLC is 8 bits and the PRESC only requires 4 bits. I think just the `uint8_t` is appropriate.

Reply:
I had to change this to 16 bits because `analogReadPrescaler` takes in the actual prescaler value (not the register value) and the value 256 doesn't fit in 8 bits.

Reply:
Got it, thanks for clarifying!